src/components/scenes/Loans/LoanCreateConfirmationScene.tsx

```diff
@@ -107,7 +105,7 @@ export const LoanCreateConfirmationScene = (props: Props) => {
 
       // Target mainnet native balance should be double the fees estimate to be
       // extra generous when accounting for fee volatility.
-      const feeNativeAmount = sub(mul(mainNetworkFeeAmount, '2'), borrowWalletNativeBalance)
+      const feeNativeAmount = sub(mul(mainNetworkFeeAmount, '1.25'), borrowWalletNativeBalance)
```

Comment:
Can we hold off on this change just because we don't want it impacting our launch. We can create a task to reconsider this post-beta?